.releaserc

```diff
@@ -1,3 +1,4 @@
 {
-    "branches": ['main']
+    "branches": ['main'],
+    "plugins": ["semantic-release-export-data"]
```

Comment:
This plugin makes it so the `semantic-release` command outputs a `new-release-version` variable in github actions that I later use to set the SDK version in the `postRequest.ts` file